main.nf

```diff
@@ -44,18 +50,21 @@ process download_fastqs {
     maxForks params.maxConcurrentDownloads
     time { 1.hour * task.attempt }
 
-    errorStrategy { task.attempt < 3 ? 'retry' : 'ignore' }   
+    errorStrategy { task.attempt < 3 && task.exitStatus != 2 ? 'retry' : 'ignore' }   
```

Comment:
Any particular reason why exit code 2 not 1 ?

Reply:
See the process code below- the exit code of 2 is used to return an error indicating local files were not found in a controlled access experiment (and retries are therefore not worthwhile).